<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>903</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Проект</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>1081</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>АРИФМЕТИКА - ЭТО ПРОСТО!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Easy">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>ЛЁГКИЙ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Hard">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>СЛОЖНЫЙ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выберите сложность:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorEasy">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ЦВЕТ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorNormal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ЦВЕТ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorHard">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ЦВЕТ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Normal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>СРЕДНИЙ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>100</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>*Тут можно передохнуть*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="KLIK">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="text">
     <string>КЛИК</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>420</y>
      <width>41</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="otvet">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>570</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ОТВЕТИТЬ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>485</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Правильные ответы:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>545</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Неправильные ответы:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kak">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kak1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
